packages/theme-toolkit/src/component-stories-candidate.tsx

```diff
@@ -0,0 +1,377 @@
+    },
+  },
+  {
+    storyId: 'react-nl-link--hover',
+    group: STORY_GROUPS['LINK'],
+    name: 'Candidate Link: Hover',
+    state: true,
+    inline: true,
+    render: () => (
+      <Link href="https://example.com" target="_new" className="utrecht-link--hover">
+        The Quick Brown Fox Jumps Over The Lazy Dog
+      </Link>
+    ),
+    detectTokens: {
+      anyOf: ['nl.link.hover.color', 'nl.link.hover.text-decoration', 'nl.link.hover.text-decoration-thickness'],
+    },
+  },
+  {
+    storyId: 'react-nl-link--placeholder',
+    group: STORY_GROUPS['LINK'],
+    name: 'Candidate Link: Placeholder',
+    state: true,
+    inline: true,
+    render: () => (
+      <Link href="https://example.com" target="_new">
+        The Quick Brown Fox Jumps Over The Lazy Dog
+      </Link>
+    ),
+    detectTokens: {
+      anyOf: ['nl.link.placeholder.color', 'nl.link.placeholder.font-weight'],
+    },
+  },
+  {
+    storyId: 'react-nl-link--visited',
+    group: STORY_GROUPS['LINK'],
+    name: 'Candidate Link: Visited',
+    state: true,
+    inline: true,
+    render: () => (
+      <Link href="https://example.com" target="_new" className="utrecht-link--visited">
+        The Quick Brown Fox Jumps Over The Lazy Dog
+      </Link>
+    ),
+    detectTokens: {
+      anyOf: ['nl.link.visited.color'],
+    },
+  },
+  {
+    storyId: 'react-nl-mark--default',
+    name: 'Candidate Mark',
+    render: () => <Mark>Mark my words!</Mark>,
+    detectTokens: {
+      anyOf: ['nl.mark.color', 'nl.mark.background-color'],
+    },
+  },
+  {
+    storyId: 'react-nl-number-badge--default',
+    group: STORY_GROUPS['BADGES'],
+    name: 'Candidate Number Badge',
+    render: () => <NumberBadge>42</NumberBadge>,
+  },
+  {
+    storyId: 'react-nl-paragraph--default',
+    group: STORY_GROUPS['PARAGRAPH'],
+    name: 'Candidate Paragraph',
+    render: () => (
+      <Paragraph>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+        laborum
+      </Paragraph>
+    ),
+    detectTokens: {
+      anyOf: [
+        'nl.paragraph.color',
+        'nl.paragraph.font-size',
+        'nl.paragraph.font-family',
+        'nl.paragraph.font-weight',
+        'nl.paragraph.line-height',
+        'nl.paragraph.margin-block-start',
+        'nl.paragraph.margin-block-end',
+      ],
+    },
+  },
+  {
+    storyId: 'react-nl-paragraph--lead',
+    component: 'nl-paragraph',
+    group: STORY_GROUPS['PARAGRAPH'],
+    name: 'Candidate Lead Paragraph',
+    render: () => (
+      <Paragraph>
```

Comment:
Ben je hier vergeten om `purpose` toe te voegen? Nu verwacht ik dat hij een gewone paragraph rendert. Ik zou ook maar 1 alinea doen, dit is wel heel uitgebreid qua Chromatic pixels :)